docker/Dockerfile

```diff
@@ -4,6 +4,10 @@ FROM socrata/java
 ENV SERVER_NAME region-coder
 ENV SERVICE_PORT 2021
 EXPOSE 2021
+ENV JMX_PORT 9999
+ENV JMX_RMI_PORT 9998
```

Comment:
Hm, that's a good idea to try and make it one port.   Also agreed with putting it in the same range as the app port.  Maybe xx99 (e.g., geospace is 2021, so the JMX port is 2099).